request-handler/src/main/java/au/org/aodn/aws/wps/operation/ExecuteOperation.java

```diff
@@ -82,11 +82,11 @@ public String execute(Properties config) {
 
         LOGGER.debug("Job submitted.  Job ID : " + jobId);
 
-        String statusLocation = statusLocationBase + "prototypeId" + "/" + statusFileName;
-        ExecuteStatusBuilder statusBuilder = new ExecuteStatusBuilder(statusLocation,jobId);
+        String statusLocation = "https://s3.amazonaws.com/" + statusS3BucketName + "/" + jobId + "/" + statusFileName;
+        ExecuteStatusBuilder statusBuilder = new ExecuteStatusBuilder(statusLocation, jobId);
         String statusDocument = statusBuilder.createResponseDocument(EnumStatus.ACCEPTED);
 
-        S3StatusUpdater statusUpdater = new S3StatusUpdater(statusLocationBase, "prototypeId");
+        S3StatusUpdater statusUpdater = new S3StatusUpdater(statusS3BucketName, jobId);
```

Comment:
Does this have code to generate the status location as well?  would it better to consolidate this into one location?